src/vue/backend/package.json

```diff
@@ -33,7 +33,7 @@
     "eslint-plugin-promise": "^4.2.1",
     "eslint-plugin-standard": "^4.0.0",
     "eslint-plugin-vue": "^6.2.2",
-    "node-sass": "^4.12.0",
+    "node-sass": "^4.14.0",
```

Comment:
Will this negatively effect people with a different node version than you?

Reply:
I'm not sure

Reply:
We should talk about this before its merged in, to avoid breaking anything unexpected